src/transactions/TxResultsTests.cpp

```diff
@@ -797,4 +798,40 @@ TEST_CASE("txresults", "[tx][txresults]")
             });
         }
     }
+
+    SECTION("fees with liabilities")
+    {
+        auto acc = root.create("acc", lm.getMinBalance(1) + baseFee + 1000);
```

Comment:
I think that edge case here is that if you added +1 to that balance, it would pass even for version 1. Maybe it is worth adding such test case for contrast?